src/Controller/PostController.php

```diff
@@ -37,15 +37,15 @@
 use Sonata\NewsBundle\Model\PostInterface;
 use Sonata\NewsBundle\Model\PostManagerInterface;
 use Sonata\SeoBundle\Seo\SeoPageInterface;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\Form\FormInterface;
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 use Symfony\Component\Security\Core\Exception\AccessDeniedException;
 
-class PostController extends Controller
+class PostController extends AbstractController
```

Comment:
We working on close API. Can you add `final` here? Also will be nice to add notice about in `3.x` like here:
https://github.com/sonata-project/SonataMediaBundle/blob/3.x/src/Controller/GalleryController.php#L21

Reply:
If you would I can make a second commit closing a little more the API in all controllers